### ISSUE TITLE
Accordion component design updates (ui kit v5)

### ISSUE THREAD

**Issue by hannahiss** [Orange-OpenSource/Orange-Boosted-Bootstrap#1791]
Implement new design
![image](https://user-images.githubusercontent.com/16681352/212877808-8ba08e2b-0d91-4219-a982-5eab5c623a9a.png)

- change background-color on hover to `#eee` (and for dark variant to `#333`)
- change active color to `black` (and for dark variant `white`)
- no change on focus: due to box-shadow, it is not possible to see gray border below (=> needs design review to confirm with the designer)
